simple/markupsafe/index.html

```diff
@@ -9,7 +9,5 @@ <h1>Links for markupsafe</h1>
     <!-- <a href="">FILENAME</a><br/> -->
     <a href="/localwheels/MarkupSafe-1.0.tar.gz">MarkupSafe-1.0.tar.gz</a><br/>
     <a href="/localwheels/MarkupSafe-1.0-cp35-cp35m-linux_x86_64.whl">MarkupSafe-1.0-cp35-cp35m-linux_x86_64.whl</a><br/>
-    <a href="/localwheels/MarkupSafe-1.1.0.tar.gz">MarkupSafe-1.1.0.tar.gz</a><br/>
```

Comment:
missing a link to `MarkupSafe-1.0-cp37-cp37m-linux_x86_64.whl` here?

Reply:
Yup, typo again, sorry. Updated the PR.